zh-CN/strings.po

```diff
@@ -2175,15 +2175,15 @@ msgstr "可交易的"
 
 #. i18n: keyword: MismatchedKuTitle
 msgid "Mismatched User IDs"
-msgstr ""
+msgstr "用户ID未找到"
```

Comment:
I think this translation might be a little bit incorrect - it should not mean "ID could not be found".
"Mismatched User IDs" can also be something like "User IDs do not match".

Reply:
Problem solved.